src/main/java/longah/node/Transaction.java

```diff
@@ -69,8 +82,26 @@ public void parseTransaction(String expression, MemberList members) throws LongA
         // Check for existence of all parties involved in the transaction in the group.
         String lenderName = splitInput[0].trim();
         this.lender = members.getMember(lenderName);
+        String borrowNameAmount;
         for (int i = 1; i < splitInput.length; i++) {
-            String borrowNameAmount = splitInput[i].trim();
+
+            if (i == splitInput.length - 1) {
+                String[] inputWithDateTime = splitInput[i].split("t/");
+                if (inputWithDateTime.length == 2) {
+                    try {
```

Comment:
Feel is a poor arrowhead implementation. Might be better to check for the existence of Time input outside the for loop using `expression.contains("t/")` and extract out that string prior to entering the for loop, and keep the for loop as is.

Reply:
Resolved!